docs/README.md

```diff
@@ -62,6 +62,23 @@ externally hosted documentation. The reference will look like `integrate_connect
 
 Replace <connector_name> with the name of your connector, e.g., `elasticsearch` for `flink-connector-elasticsearch`.
 
+### Include external documentation hosted on forks
+
+The configuration above works for connectors already merged in `apache/flinl-connector-<connector-name>` repos.
+When contributing to a Flink's connector, the documentation is hosted in a fork of one of these repos.
```

Comment:
I am not sure what this means. Do you mean the docs reside alongside the connector code in a separate repo? They are not _hosted_ in a fork, they are pulled in at _build_ time

Reply:
I probably need to rephrase. The key is that the script generating docs only pulls branch/tags from the official repos. If you are contributing to a connector and you want to preview your changes before they are merged, you must include what is in your fork